corehq/apps/app_manager/management/commands/flag_domains_using_search_filter.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import BaseCommand
+
+from corehq import toggles
+from corehq.apps.app_manager.dbaccessors import domain_has_apps, get_app_ids_in_domain, get_current_app
+from corehq.util.log import with_progress_bar
+
+
+class Command(BaseCommand):
+    help = "One-time command to enable search filter flag where 'search filter' is already in use"
+
+    def handle(self, **options):
+        flag_domains_using_search_filter()
+
+
+def flag_domains_using_search_filter():
+    domains = (
+        set(toggles.USH_CASE_CLAIM_UPDATES.get_enabled_domains())
+        - set(toggles.USH_SEARCH_FILTER.get_enabled_domains())
```

Comment:
As we discussed earlier, I wonder if it makes sense to remove this section that checks for domains with the new toggle `USH_SEARCH_FILTER` enabled. There will be none when this command is run and the chances of a partial migration failure seem small.